app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlView.kt

```diff
@@ -205,6 +206,13 @@ class SessionControlView(
                     super.onLayoutCompleted(state)
 
                     JumpBackInCFRDialog(view).showIfNeeded()
+                    val syncCFRPresenter = SyncCFRPresenter(
```

Comment:
Since we're not using the `syncCFRPresenter` anywhere else, should we just call `SyncCFRPresenter(...).start()`?